src/drivers/vue.js

```diff
@@ -0,0 +1,46 @@
+import { extend } from '../lib';
+
+
+export let vue = {
+
+
+    global() {
+
+    },
+
+    register(component) {
+
+        return {
```

Comment:
Maybe we should do `Vue.component('my-component', { .. });` and register this as a global component, using the `component.tag` as the name?